py/emitbc.c

```diff
@@ -36,7 +36,7 @@
 
 #if MICROPY_ENABLE_COMPILER
 
-#define BYTES_FOR_INT ((BYTES_PER_WORD * 8 + 6) / 7)
+#define BYTES_FOR_INT ((MP_BYTES_PER_OBJ_WORD * 8 + 6) / 7)
```

Comment:
So the idea is that while global definitions should all have the `MP_` prefix, definitions local to a file can omit that for brevity?

Reply:
Yeah that seems to be the way here.  I mean, there are a lot of internal functions that don't have an `mp_` prefix either.

We could eventually change that, but for now I was just concentrating on the ones in `py/mpconfig.h`.

Reply:
I think it's ok to leave it as-is, was just checking that this is what the situation is like currently.

Reply:
Ok, cool.